src/components/CommentsByArticleId.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react'
+import axios from 'axios';
+import CommentCard from './CommentCard';
+export default function CommentsByArticleId({id, activeUserName}){
+    let articleId = id;
+    const [commentId, setCommentId] = useState(null)
+    const [commentsVote, setCommentsVote] = useState(null)
+    const [commentsList, setCommentsList] = useState([])
+
+    useEffect(()=>{
+        axios.get(`https://nc-news-24h6.onrender.com/api/articles/${id}/comments`).then((response)=>{
```

Comment:
It would be best to separate out your api functions into a util file, you could then export these functions where needed in your app. It would separate out your concerns and mean you only need to import axios in one file.